gateway/src/shard/builder.rs

```diff
@@ -266,7 +266,7 @@ impl ShardBuilder {
 
     /// Set the maximum number of members in a guild to load the member list.
     ///
-    /// Default value is `250`. The minimum value is `50` and the maximum is
+    /// Default value is `50`. The minimum value is `50` and the maximum is
```

Comment:
it's 250 https://github.com/twilight-rs/twilight/blob/28081dd660e9fef9a18c482cec89815b2421f8c8/gateway/src/shard/builder.rs#L185

Reply:
So shouldn't we update that value too?

Reply:
If that's the purpose of this PR, yes